.github/workflows/publish-commit-pkg.yml

```diff
@@ -1,14 +1,16 @@
-name: Publish Approved Pull Requests Pkg
+name: Publish Commit Pkg
 
 on:
-  pull_request_review:
-    types: [submitted]
+  push:
+    branches:
+      - dev
+    tags:
+      - '!**'
```

Comment:
It could be adjusted to the initial state, executed after the pr is created and the document is built so the creator can test it himself. 🤔

Reply:
After building the document, execute ci. The pr.new.pkg bot will not comment, so I write it directly as pr execution.

Reply:
> It could be adjusted to the initial state, executed after the pr is created and the document is built so the creator can test it himself. 🤔

I tested two PRs and asked a warehouse to install and download them. I found that the execution results of the robot's initial comment address were different in different environments. I don't know where the problem is.
So now I think it's best not to let the robot comment, and do it according to what you said above

https://github.com/warmthsea/elp-pkg-pr-release-test

https://github.com/warmthsea/element-plus/pull/20
https://github.com/warmthsea/element-plus/pull/21